nemo/collections/nlp/models/machine_translation/mt_enc_dec_model.py

```diff
@@ -365,6 +365,19 @@ def eval_epoch_end(self, outputs, mode):
                 self.log(f"{mode}_sacreBLEU_dl_index_{dataloader_idx}", sb_score, sync_dist=True)
                 getattr(self, f'{mode}_loss_{dataloader_idx}').reset()
 
+            # add logs if available
+            log_dict = {}
+            for x in output:
+                if "log" in x:
+                    for k, v in x["log"].items():
+                        log_dict[k] = log_dict.get(k, []) + [v]
+
+            for k, v in log_dict.items():
+                if dataloader_idx == 0:
+                    self.log(f"{mode}_{k}", np.mean(v), sync_dist=True)
+                else:
+                    self.log(f"{mode}_{k}_dl_index_{dataloader_idx}", np.mean(v), sync_dist=True)
+
```

Comment:
What's being logged here? when is `"log" in x` ?

Reply:
I can only find NeMo/examples/nlp/language_modeling/get_wkt2.sh which is not empty (and was not added by me). Could you provide a full path for the file?

Reply:
I think the issue with get_wkt2.sh is not that it is an empty file, but for some reason, the permissions on that file were changed.

Reply:
From 100644 → 100755. chmod 644 should change it back.